diaspora_federation-tests.gemspec

```diff
@@ -0,0 +1,24 @@
+$LOAD_PATH.push File.expand_path("../lib", __FILE__)
+
+# Maintain your gem's version:
+require "diaspora_federation/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = "diaspora_federation-tests"
+  s.version     = DiasporaFederation::VERSION
+  s.authors     = ["Benjamin Neff"]
+  s.email       = ["benjamin@coding4.coffee"]
+  s.homepage    = "https://github.com/SuperTux88/diaspora_federation"
+  s.summary     = "diaspora* federation library"
+  s.description = "This gem provides some supplimentary code (factory definitions), that"\
+                  "help to build tests for users of the diaspora_federation gem."
+  s.license     = "AGPL 3.0 - http://www.gnu.org/licenses/agpl-3.0.html"
+
+  s.files       = Dir["lib/diaspora_federation/factories.rb"]
```

Comment:
then maybe name the gem "diaspora_federation-test"?

I don't mean the "test" folder at toplevel, I mean "lib/diaspora_federation/test/factories.rb". You should also add a "lib/diaspora_federation/test.rb" as entry point for the gem. (see "lib/diaspora_federation/rails.rb" for the -rails gem)